src/dotnet/project.json

```diff
@@ -24,7 +24,7 @@
         "Microsoft.CodeAnalysis.CSharp":  "1.3.0-beta1-20160225-02",
         "Microsoft.DiaSymReader.Native": "1.3.3",
 
-        "NuGet.CommandLine.XPlat": "3.4.0-beta-632",
+        "NuGet.CommandLine.XPlat": "3.4.0-beta-680",
```

Comment:
There is a reference to NuGet here as well: https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.ProjectModel/project.json#L9